src/Reusables/Navbar.js

```diff
@@ -0,0 +1,33 @@
+
+function Navbar() {
+    return (
+        <header class="nav-header">
+        <div class="nav-header-div">
+            <div class="nav-header-div-lt">
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#top">HOME</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#top">FILTER</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#top">PRODUCTS</a>
+            </div>
+            <div class="nav-header-div-mid">
+                <input class="input-outlined" type="text" placeholder="Search" />
+            </div>
+            <div class="nav-header-div-rt">   
+                        <a href="#top" class="btn-small margin-lr">Login</a>
+
+                        <div class="badge-container margin-lr">
+                            <a href="#top"><img
```

Comment:
I think you forgot to convert class to className

Reply:
yes i replaced them, thanks